ocs_ci/ocs/ocp.py

```diff
@@ -401,3 +401,64 @@ def switch_to_default_rook_cluster_project():
         bool: True on success, False otherwise
     """
     return switch_to_project(defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def get_default_storage_class():
+    """
+    Get the default StorageClass(es)
+
+    Returns:
+        list: StorageClass(es) list
+
+    """
+    count = 0
```

Comment:
Let's return the list of default storageclasses and examine it from the test. 
if we are expecting 1 default SC, then we can assert it from outside:

```python
assert len(default_storage_classes) > 1, "More than 1 storage class exist"
```

Reply:
Done